src/extension/background-script/events/notifications.js

```diff
@@ -6,11 +6,18 @@ const paymentSuccessNotification = (message, data) => {
   const recipient = data.origin.name;
   const paymentResponse = data.response;
   const route = paymentResponse.data.route;
-  const { total_amt } = route;
+  const { total_amt, total_fees } = route;
+  const paymentAmount = total_amt - total_fees;
+
+  function formatAmount(amount) {
```

Comment:
This possibly opens a whole new discussion. 😅 What is the plural of 1 sat? 
 - 2 sat
 - 2 sats
 
 To me sats feel a little bit more natural but I have no strong opinion on that.

Reply:
hehe. yes. is it sat or sats that's the question. maybe we should do a twitter poll?

Reply:
While drafting on the poll I found a very promising definition. There are basically 2 types of usages:

 - Unit (an amount of money, i.e. 99 satoshi, cent, etc) "please pay this invoice of 100 satoshi"
 - Objects (coins, i.e. 99 satoshis, cents, etc) "stacking sats"
 
 Since I feel we use it more like a unit I would vote for against using `sats`. 🤔

Reply:
this is what twitter says: https://twitter.com/getAlby/status/1509884716517928963

Reply:
That's a clear result. Let's keep the `s`! 😀